config/default/manager_metrics_patch.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
```

Comment:
I think they should be in `manager.yaml` and not exist as a patch since `controller-manager-metrics-service` is provided in `manager.yaml`

Reply:
Maybe this is designed as patch is better than hard code in manager.yaml.

Reply:
I don't understand why it could be better

Reply:
I think if `controller-manager-metrics-service` is hard-coded in `manager.yaml`, then the relevant container should also be in `manager.yaml`, or they should all be in the patch